File: implementations/node/sdk/src/contracts/publication/publication-dependency-package.ts
import type { AiSdkPublicationDependencyPackageKind } from "./publication-dependency-package-kind.js";

export interface AiSdkPublicationDependencyPackage {
  readonly schemaVersion: number;
  readonly kind: AiSdkPublicationDependencyPackageKind;
  readonly manifestPath: string;
}

File: implementations/python/sdk/src/multiplexed_ai_sdk/contracts/publication/publication_dependency_package.py
from dataclasses import dataclass

from ...wire_model import AiSdkWireModel
from .publication_dependency_package_kind import AiSdkPublicationDependencyPackageKind


@dataclass(frozen=True)
class AiSdkPublicationDependencyPackage(AiSdkWireModel):
    schema_version: int
    kind: AiSdkPublicationDependencyPackageKind
    manifest_path: str
